addons/docs/src/frameworks/svelte/HOC.svelte

```diff
@@ -15,4 +15,5 @@
   });
 </script>
 
+<svelte:options accessors={true} />
```

Comment:
what does this do?

Reply:
That option allows getters and setters on the svelte component, i need this option on to extract the injected __docgen prop inside each svelte component
```ts
const docs = comp.__docgen;
```
you can view the svelte:options API here: https://svelte.dev/docs#svelte_options
```
accessors={true} — adds getters and setters for the component's props
```